test/integration/basic_test.go

```diff
@@ -152,6 +152,7 @@ func (m *Minishift) commandReturnShouldBeEmpty(commandField string) error {
 func FeatureContext(s *godog.Suite) {
 	var givenArgs = flag.String("minishift-args", "", "Arguments to pass to minishift")
 	var givenPath = flag.String("binary", fmt.Sprintf("../../out/%s-amd64/minishift", runtime.GOOS), "Path to minishift binary")
```

Comment:
@hferentschik just realized that make file and basic_test.go are in different folders, so there are troubles with using relative path. basic_test.go needs to have `../../out/linux-amd64/minishift` and make file needs `./out/linux-amd64/minishift`. Working solution for me is to define the path as absolute path by prepending $GOPATH: `MINISHIFT_BINARY ?= $(GOPATH)/src/$(REPOPATH)/out/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)`